src/moodleapi/Moodleapi.ts

```diff
@@ -0,0 +1,39 @@
+interface Response {
+  uuid: string
+  jwt: string
+}
+export class Moodleapi {
+  endpoint = 'local_raise_get_user'
+
+  async getUser(): Promise<Response> {
+    const userRequest = {
+      index: 0,
+      methodname: this.endpoint,
+      args: {
+      }
+    }
+    const url = this.getMoodlePluginMethodUrl(this.endpoint)
+    if (url === '') {
+      console.log('Invalid environment variables')
+    }
```

Comment:
This logic can be removed once we incorporate passing the `wwwroot` and `sesskey` in a constructor.